modules/dod/README.md

```diff
@@ -4,11 +4,23 @@
 
 Domains of Delight are to the Feywild what Domains of Dread are to the Shadowfell: sequestered realms governed by powerful beings. Whereas a Domain of Dread is ruled by a Darklord, a Domain of Delight is ruled by an archfey—the most powerful of Fey creatures. An archfey gives form to their Domain of Delight, shaping it in ways unique to their personality. Some Domains of Delight are bright and cheery, while others are gloomy, but each one reflects the emotional state of its ruler. A Domain of Delight can be as small as a few acres or as big as a country.
 
-This accessory helps you create Domains of Delight and the archfey who rule them, building on the information about the Feywild that appears in the _Dungeon Master's Guide_. The ideas, tips, and tables in this chapter are meant to spark your imagination. Use what excites and intrigues you, discard what doesn’t, and make up the rest!
+This accessory helps you create Domains of Delight and the archfey who rule them, building on the information about the Feywild that appears in the **. The ideas, tips, and tables in this chapter are meant to spark your imagination. Use what excites and intrigues you, discard what doesn’t, and make up the rest!
 
 - **Adds Detailed information and tips about the Feywild** and how to use it in your game
-- Gives DMs **everything they need to create their own Archfey and Domains of Delight**
+- Gives DMs **everything they need to create their own Archfey and Domains of Delight**
+
+Discover the other books benefiting Extra Life:
 
+- 
+- 
+- 
+- 
+- 
+
+---
+
+Extra Life supports Children’s Miracle Network Hospitals® by raising funds and awareness that provide 32 million treatments each year to kids across the U.S. and Canada.
+

```

Comment:
Empty list.

Not sure we want to keep the Extra Life statement? We remove the plug in this spot for most books, but being for a charity I'd be alright with keeping this one.